element_array_ephys/ephys_organoids.py

```diff
@@ -128,382 +130,167 @@ class AcquisitionSoftware(dj.Lookup):
     definition = """  # Software used for recording of neuropixels probes
     acq_software: varchar(24)
     """
-    contents = zip(["SpikeGLX", "Open Ephys"])
+    contents = zip(["SpikeGLX", "Open Ephys", "Intan"])
 
 
 @schema
-class ProbeInsertion(dj.Manual):
-    """Information about probe insertion across subjects and sessions.
-
-    Attributes:
-        Session (foreign key): Session primary key.
-        insertion_number (foreign key, str): Unique insertion number for each probe insertion for a given session.
-        probe.Probe (str): probe.Probe primary key.
-    """
+class EphysSession(dj.Manual):
+    """User defined ephys session for downstream analysis."""
 
     definition = """
-    # Probe insertion chronically implanted into an animal.
     -> Subject
-    insertion_number: tinyint unsigned
+    -> probe.ElectrodeConfig 
```

Comment:
From our team discussion, I recall us considering making this a secondary attribute to avoid a UUID being in the primary key.  What do you think?